### ISSUE TITLE
Tooltip is not defined for ‘Close (X) icon’ button on ‘Pages’ popup.

### ISSUE THREAD

**Issue by azaslonov** [paperbits/paperbits-demo#93]
User Experience:

If user can not perceive the sensory characteristics of components such as shape, size, visual location, orientation, or sound, and no additional information is provided then the people with  disabilities and cognitive limitations will not be access the instructions for using the content.

Note: User credentials should NOT be included in the bug.

 

Repro Steps:

1. Open Azure portal with valid credentials and create a API Management.

2. Click Developer Portal button to navigate to the portal.

3. Wait portal to load in design mode.

4. Navigate to the Left nav and select Pages button.

5. Navigate to the various controls of the "Pages" window.

6. Open Developer tool using F12 key and navigate to ‘Close’ icon button.

7. Verify whether tooltip is defined or not for ‘Close’ icon button.

                                  

Actual Result:

Tooltip is not defined for ‘Close (X) icon’ button, present on ‘Pages’ popup.

 

Expected result:

Tooltip should be defined for ‘Close (X) icon’ buttons, present on ‘Pages’ popup.

Note:

Similar issue exist with Chrome.

Similar issue exist throughout the application.

 

Reference Links

External Bug Process: If this bug belongs to external team, mark it as resolved/done and assign it back to the tester with notes on where to file/route the bug. For more information please use this Link. “HCL Staff should not log any third party external bugs. Should be routed to edad team”.
Please reach out to C&AI Teams channel for any process related queries 
 
![image](https://user-images.githubusercontent.com/2320302/77595472-7983b780-6eb6-11ea-87c6-9aaa45b91385.png)
